lib/src/main/java/com/basistheory/android/view/mask/MaskResult.kt

```diff
@@ -1,17 +0,0 @@
-package com.basistheory.android.view.mask
-
-import android.text.Editable
-
-internal data class MaskResult(
-    val maskedValue: String,
-    val unMaskedValue: String,
```

Comment:
i guess the unmasked value is not needed cuz we already have it on the element?

Reply:
Correct, and it was never used